src/modules/ShopgateExecuteWithRetry.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Creates an interval and executes the executeCallback until it returns true or interval finishes
+ *
+ * @param {int} intervalInMiliseconds interval in milisenconds
+ * @param {int} maximumIntervallTimeInMiliseconds maximum interval in miliseconds
+ * @param {function} executeCallback callback has to return true in case it will finish execution
+ *                                   and false if it should repeatedly executed
+ */
+export function shopgateExecuteWithRetry(
```

Comment:
See how a call actually looks:

  shopgateExecuteWithRetry(25, 2500, function() {
    if (!window.SGJavascriptBridge) {
      return false;
    }

    shopgateInit();

    return true;
  })

after your suggested change:

  shopgateExecuteWithRetry(function() {
    if (!window.SGJavascriptBridge) {
      return false;
    }

    shopgateInit();

    return true;
  }, 25, 2500)

I prefer clearly the first because it makes it harder for the reader of the code to find the other parameters. My example is just a small example the code could be more complex.